android/src/test/java/com/onegini/mobile/sdk/reactnative/Utils.kt

```diff
@@ -0,0 +1,58 @@
+package com.onegini.mobile.sdk.reactnative
+
+import com.facebook.react.bridge.Promise
+import com.facebook.react.bridge.ReadableArray
+import com.facebook.react.bridge.ReadableMap
+import com.onegini.mobile.sdk.android.client.OneginiClient
+import com.onegini.mobile.sdk.android.client.OneginiClientBuilder
+import com.onegini.mobile.sdk.android.handlers.OneginiInitializationHandler
+import io.mockk.*
+import java.util.*
+import kotlin.collections.HashMap
+import kotlin.collections.set
+
+object Utils {
+
+    fun createProvider(id: String, isTwoStep: Boolean): HashMap<String, *> {
+        val map = HashMap<String, Any>()
+        map["id"] = id
+        map["isTwoStep"] = isTwoStep
+        return map
+    }
+
+    fun startClient(rNOneginiSdk: RNOneginiSdk, oneginiClient: OneginiClient, arrayProviders: ArrayList<Any>?) {
```

Comment:
In Kotlin you can set default values for parameters so there's no need for this extra function. 
```
    fun startClient(rNOneginiSdk: RNOneginiSdk, oneginiClient: OneginiClient, arrayProviders: ArrayList<Any>?, enableFingerprint: Boolean = false) {
    ...
```